src/services/recim/series.ts

```diff
@@ -81,6 +81,11 @@ export type BracketInfo = {
   IsLosers: boolean;
 };
 
+type AutoScheduleParameters = {
+  RoundRobinMatchCapacity: number;
+  NumberOfLadderMatches: number;
+};
```

Comment:
This is not accurate to the data you're actually passing.
```suggestion
type AutoScheduleParameters = {
  RoundRobinMatchCapacity: number;
} | {
  NumberOfLadderMatches: number;
};
```

Reply:
was unaware I could do this. I just fully expected `null` to translate. But it makes sense.